lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,158 @@
+.container {
+    background-color: $color-bg-white;
+    display: inline-flex;
+    flex-direction: column;
+    margin: 10px 40px;
+
+    button,
+    a {
+        background: $color-bg-white;
+    }
+
+    svg {
+        color: $color-text-blue-dark;
+    }
+
+    button:focus,
+    button:hover,
+    a:focus,
+    a:hover {
+        background: $color-bg-gray-light;
+        color: $color-link-black;
+    }
+
+    div:not(.resource-edit) {
+        display: flex;
```

Comment:
This has still not gone away, so the Subjects widget is still broken.

Reply:
Ok, flex is now removed.